src/app/components/CurrencyAsset/index.module.css

```
@@ -0,0 +1,7 @@
+.small {
+    font-family: "Montserrat", sans-serif;
+    font-weight: 100;
+    font-size: 0.75em;
+    opacity: 50%;
+    letter-spacing: 0;
+  }
```

Import only styles.symbol to parent span element to prevent duplicated imports in symbolMap.

We only need to change font-size and opacity, all other properties should be inherited from parent.
opacity: 50% is being transformed to 1% for some reason (maybe webpack issue) so use 0.5 instead.

All files must end with newline.

```suggestion
.symbol small {
    font-size: 0.75em;
    opacity: 0.5;
}

```